### PR TITLE
[Snyk] Fix for 3 vulnerabilities

### PR DESCRIPTION
**Pull request by MelvinBot**
### Details

    

- Changes to the following files to upgrade the vulnerable dependencies to a fixed version:
    - package.json
    - package-lock.json

##### With an upgrade:
Severity                   |  Issue                   | Breaking Change                   | Exploit Maturity
:-------------------------:|:-------------------------|:-------------------------|:-------------------------
high severity | Heap-based Buffer Overflow <br/>[SNYK-JS-ELECTRON-5710623](https://snyk.io/vuln/SNYK-JS-ELECTRON-5710623) |  No  | No Known Exploit 
high severity | Use After Free <br/>[SNYK-JS-ELECTRON-5710626](https://snyk.io/vuln/SNYK-JS-ELECTRON-5710626) |  No  | No Known Exploit 
medium severity | Regular Expression Denial of Service (ReDoS) <br/>[SNYK-JS-SEMVER-3247795](https://snyk.io/vuln/SNYK-JS-SEMVER-3247795) |  Yes  | Proof of Concept 

<details>
  <summary><b>Commit messages</b></summary>
  </br>
  <details>
    <summary>Package name: <b>@storybook/addon-essentials</b></summary>
    The new version differs by 250 commits.</br>
    <ul>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/4f2afa644d7f2833181fc03187f5597d442285a6">4f2afa6</a> v7.0.0</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/03292b004b07b55066ac95ae35e29bba9d169108">03292b0</a> Update root, peer deps, version.ts/json to 7.0.0 [ci skip]</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/b2dc5cf6757aa98f02f057fd1ab6df48aaefbc8f">b2dc5cf</a> Revert &quot;Update root, peer deps, version.ts/json to 7.0.0 [ci skip]&quot;</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/7f391a3894b509ab4bb331a0546e5362eaa3e2b3">7f391a3</a> Update root, peer deps, version.ts/json to 7.0.0 [ci skip]</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/f0b53cb9b0991050b552cec6a63969ffdb1f3e98">f0b53cb</a> 7.0.0 changelog</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/930917d9c5f3726c69ee82e6a744aa40a5e061a8">930917d</a> Merge pull request #21856 from storybookjs/docs/interactions-addon-migration</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/f1c13da545524e186d798b53dcc718ae32eaf79e">f1c13da</a> 7.0.0-rc.11 next.json version file [skip ci]</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/512a2ae5a65192cb01fc8a9baf0025ae0f59cf57">512a2ae</a> Update git head to 7.0.0-rc.11, update yarn.lock [ci skip]</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/908c324acf53fc0f97fbdd1ff14d39a5dde396ff">908c324</a> v7.0.0-rc.11</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/5edc7c0f4eca2974b137296001eed49d1cb1eeef">5edc7c0</a> Update root, peer deps, version.ts/json to 7.0.0-rc.11 [ci skip]</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/324d9bbb923ad51bcea89cf8b3d7849a0105072c">324d9bb</a> 7.0.0-rc.11 changelog</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/37d9737a89932e89a85b31d06559dcb94ea122fc">37d9737</a> interactions debugger is now default</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/9682f7ce6ed708b7a5c850987ab44df688ac91a0">9682f7c</a> Merge pull request #21833 from storybookjs/kasper/fix-strict-args-decorator-with-interface</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/a08ffc781696eb89bd1b55f45998b99a816bc699">a08ffc7</a> Put @ storybook/csf version back into next</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/2cc1d36a576cccfaeb59c2470da81794f9c7c365">2cc1d36</a> Merge pull request #21850 from storybookjs/fix/tone-down-dependency-alerts</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/941103bcc1027163fb05fb540bb36805cd4247ec">941103b</a> Merge pull request #21851 from storybookjs/valentin/export-application-config-decorator</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/31700c09c2097e8d841a81c54a425e4f748e6f37">31700c0</a> Export applicationConfig decorator and adjust documentation for usage</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/3d9544f86145d2b9bf66daac4bf283316ea10853">3d9544f</a> Merge pull request #21846 from storybookjs/chore_docs_webpack_tweaks</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/79b590b8de663ad0280f198706f26346de4ddcb1">79b590b</a> Tweaks to the Webpack docs</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/d193be57c8694169e4111c258c673ee1ccf4376a">d193be5</a> Merge pull request #21836 from storybookjs/fix/downgrade-remark-deps</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/79b1fde44cd4f4ba7c835819dfc0777666dbc563">79b1fde</a> Merge pull request #21832 from storybookjs/fix/polyfill-global</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/590f053ef606d4191fc74cb893c56776dc88d302">590f053</a> downgrade remark related dependencies</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/b421d9545d2073f0a2a8fadb12f4935afea714ea">b421d95</a> only provide critical duplicated dependency warning on major version difference</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/acace304016ac30d3aa2277e2397bf55cf811337">acace30</a> Merge pull request #21724 from jungpaeng/docs/fix-controls</li>
    </ul>

   <a href="https://snyk.io/redirect/github/storybookjs/storybook/compare/893881ff6f87272c0ab0f21776ef8fef990d0c36...4f2afa644d7f2833181fc03187f5597d442285a6">See the full diff</a>
  </details>
  <details>
    <summary>Package name: <b>@storybook/react</b></summary>
    The new version differs by 250 commits.</br>
    <ul>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/4f2afa644d7f2833181fc03187f5597d442285a6">4f2afa6</a> v7.0.0</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/03292b004b07b55066ac95ae35e29bba9d169108">03292b0</a> Update root, peer deps, version.ts/json to 7.0.0 [ci skip]</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/b2dc5cf6757aa98f02f057fd1ab6df48aaefbc8f">b2dc5cf</a> Revert &quot;Update root, peer deps, version.ts/json to 7.0.0 [ci skip]&quot;</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/7f391a3894b509ab4bb331a0546e5362eaa3e2b3">7f391a3</a> Update root, peer deps, version.ts/json to 7.0.0 [ci skip]</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/f0b53cb9b0991050b552cec6a63969ffdb1f3e98">f0b53cb</a> 7.0.0 changelog</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/930917d9c5f3726c69ee82e6a744aa40a5e061a8">930917d</a> Merge pull request #21856 from storybookjs/docs/interactions-addon-migration</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/f1c13da545524e186d798b53dcc718ae32eaf79e">f1c13da</a> 7.0.0-rc.11 next.json version file [skip ci]</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/512a2ae5a65192cb01fc8a9baf0025ae0f59cf57">512a2ae</a> Update git head to 7.0.0-rc.11, update yarn.lock [ci skip]</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/908c324acf53fc0f97fbdd1ff14d39a5dde396ff">908c324</a> v7.0.0-rc.11</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/5edc7c0f4eca2974b137296001eed49d1cb1eeef">5edc7c0</a> Update root, peer deps, version.ts/json to 7.0.0-rc.11 [ci skip]</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/324d9bbb923ad51bcea89cf8b3d7849a0105072c">324d9bb</a> 7.0.0-rc.11 changelog</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/37d9737a89932e89a85b31d06559dcb94ea122fc">37d9737</a> interactions debugger is now default</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/9682f7ce6ed708b7a5c850987ab44df688ac91a0">9682f7c</a> Merge pull request #21833 from storybookjs/kasper/fix-strict-args-decorator-with-interface</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/a08ffc781696eb89bd1b55f45998b99a816bc699">a08ffc7</a> Put @ storybook/csf version back into next</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/2cc1d36a576cccfaeb59c2470da81794f9c7c365">2cc1d36</a> Merge pull request #21850 from storybookjs/fix/tone-down-dependency-alerts</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/941103bcc1027163fb05fb540bb36805cd4247ec">941103b</a> Merge pull request #21851 from storybookjs/valentin/export-application-config-decorator</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/31700c09c2097e8d841a81c54a425e4f748e6f37">31700c0</a> Export applicationConfig decorator and adjust documentation for usage</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/3d9544f86145d2b9bf66daac4bf283316ea10853">3d9544f</a> Merge pull request #21846 from storybookjs/chore_docs_webpack_tweaks</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/79b590b8de663ad0280f198706f26346de4ddcb1">79b590b</a> Tweaks to the Webpack docs</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/d193be57c8694169e4111c258c673ee1ccf4376a">d193be5</a> Merge pull request #21836 from storybookjs/fix/downgrade-remark-deps</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/79b1fde44cd4f4ba7c835819dfc0777666dbc563">79b1fde</a> Merge pull request #21832 from storybookjs/fix/polyfill-global</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/590f053ef606d4191fc74cb893c56776dc88d302">590f053</a> downgrade remark related dependencies</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/b421d9545d2073f0a2a8fadb12f4935afea714ea">b421d95</a> only provide critical duplicated dependency warning on major version difference</li>
      <li><a href="https://snyk.io/redirect/github/storybookjs/storybook/commit/acace304016ac30d3aa2277e2397bf55cf811337">acace30</a> Merge pull request #21724 from jungpaeng/docs/fix-controls</li>
    </ul>

   <a href="https://snyk.io/redirect/github/storybookjs/storybook/compare/893881ff6f87272c0ab0f21776ef8fef990d0c36...4f2afa644d7f2833181fc03187f5597d442285a6">See the full diff</a>
  </details>
  <details>
    <summary>Package name: <b>copy-webpack-plugin</b></summary>
    The new version differs by 4 commits.</br>
    <ul>
      <li><a href="https://snyk.io/redirect/github/webpack-contrib/copy-webpack-plugin/commit/46af20a94e66115d551fed65fd745e4cd8d42aee">46af20a</a> chore(release): 7.0.0</li>
      <li><a href="https://snyk.io/redirect/github/webpack-contrib/copy-webpack-plugin/commit/5d5635f0b8572724c0f8daa83bf1abf17a78429c">5d5635f</a> refactor: code (#567)</li>
      <li><a href="https://snyk.io/redirect/github/webpack-contrib/copy-webpack-plugin/commit/c6f68a53c3bc20a1985d39fa23ab2ccad4472aef">c6f68a5</a> refactor: code</li>
      <li><a href="https://snyk.io/redirect/github/webpack-contrib/copy-webpack-plugin/commit/4cea28b5f93cc570dbc42b6c10f869e221bbe16c">4cea28b</a> refactor: next</li>
    </ul>

   <a href="https://snyk.io/redirect/github/webpack-contrib/copy-webpack-plugin/compare/1eceb4bbf906996cb932a532a708c2af50134426...46af20a94e66115d551fed65fd745e4cd8d42aee">See the full diff</a>
  </details>
</details>

Check the changes in this PR to ensure they won't cause issues with your project.

------------

### Fixed Issues

$ https://github.com/Expensify/App/issues/21426
PROPOSAL: https://github.com/Expensify/App/issues/21426_ISSUE(COMMENT)

### Tests

- [x] Verify that no errors appear in the JS console

### QA Steps

- [x] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

</details>

<details>
<summary>Mobile Web - Chrome</summary>

</details>

<details>
<summary>Mobile Web - Safari</summary>

</details>

<details>
<summary>Desktop</summary>

</details>

<details>
<summary>iOS</summary>

</details>

<details>
<summary>Android</summary>

</details>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -169,18 +169,18 @@
     "ajv-cli": "^5.0.0",
     "babel-eslint": "^10.1.0",
     "babel-jest": "29.4.1",
-    "babel-loader": "^8.1.0",
+    "babel-loader": "^9.0.0",
```
Comment (aldo-expensify) [inline package.json:172]: I don't see breaking changes: https://github.com/babel/babel-loader/releases

**Comment by aldo-expensify** [comment]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>



</details>

<details>
<summary>Mobile Web - Chrome</summary>



</details>

<details>
<summary>Mobile Web - Safari</summary>



</details>

<details>
<summary>Desktop</summary>



</details>

<details>
<summary>iOS</summary>



</details>

<details>
<summary>Android</summary>



</details>

**Review by aldo-expensify — APPROVED** [review]
Seems to work fine:

### Electron

image

### Web

image


### IOS

image

### Android

image

**Comment by aldo-expensify** [comment]
ahhh story books are failing

**Comment by aldo-expensify** [comment]
I have been fighting with `storybook` and I haven't been able to make it work.

It is currently failing because `start-storybook` the binary was removed after 6.5 (or 6.4? 🤷 ). Anyway, I read that the recommendation was to use the binary `sb`, but that is not found either. I tried also with `storybook` and not found again.

I ended up upgrading more storybook to `7.0.0` using `npx storybook@next automigrate`. This update storybook but also threw some error as part of the migration.

After this, I'm able to update the script `storybook` to use the `storybook` binary (same name) like this:

image

Now, I'm stuck with the error:

image

If I remove the `-p 6006` it errors saying that it doesn't know about the option `dev` 🤦 .

**Comment by aldo-expensify** [comment]
> This is similar to #21154. I had the same issue while testing.

true!

**Review by aldo-expensify — CHANGES_REQUESTED** [review]
Storybook changes not working

**Comment by pecanoro** [comment]
These versions probably got updated in more recent PRs since it has been opened since June, so I am going to close it

**Pull request closed without merging**